Assets/BlockDiceHandler.cs

```diff
@@ -1,32 +1,32 @@
 ﻿using Fumbbl;
 using Fumbbl.View;
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
-using Fumbbl.Model;
 using static Fumbbl.Model.Types.BlockDie;
 
 public class BlockDiceHandler : MonoBehaviour
 {
+    private ViewObjectList<BlockDie> BlockDice;
+    private static Vector2 FullSize = new Vector2(40, 40);
+    private static Vector2 SmallSize = new Vector2(30, 30);
```

Comment:
If you are going to organize fields and properties, public before private is the way to go.